proto/proto/block_header.proto

```diff
@@ -4,24 +4,24 @@ package block_header;
 import "digest.proto";
 
 message BlockHeader {
-    /// the hash of the previous blocks header.
```

Comment:
protobuf's commets are just two forward slashes. This was translated by prost as:

```
/// / the hash of the previous blocks header.
```

Because the forward slash was considered part of the comment